cpp/ql/src/semmle/code/cpp/commons/Assertions.qll

```diff
@@ -69,5 +58,7 @@ class RecoverableAssert extends MacroInvocation, Assertion {
     not result.(Literal).getUnspecifiedType().(ArrayType).getBaseType() instanceof CharType
   }
 }
+/*
+ * More assertion definitions go here.
+ */
```

Comment:
(do we even want this comment?  If this file is incomplete we should finish it, or file a bug)

Reply:
You're right. It's pointless.